service.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  name: 'segment-volume-lambda',
+  resourceName: 'segment-volume-lambda',
+  deployAs: 'lambda',
+  backendBasePath: '/segment-volume-lambda',
+  port: 7785,
+  app: {
+    script: 'yarn dev',
+    watch: 'src',
+    wait_ready: false,
+  },
+};
```

Comment:
I think this has been copy and pasted at some point from our `frontend-services` repo, to the whatever lambda, and then the whatever else lambda, and then this. It's a file we use for [PM2](https://pm2.keymetrics.io/), which as far as I know is only used in that repo, I'm pretty sure it does nothing here!